components/eamxx/src/physics/share/physics_test_data.hpp

```diff
@@ -70,7 +79,16 @@ struct SHOCGridData : public PhysicsTestData {
 #define  PTD_ASS8(a, b, c, d, e, f, g, h          )  PTD_ASS7(a, b, c, d, e, f, g)          ; h = rhs.h
 #define  PTD_ASS9(a, b, c, d, e, f, g, h, i       )  PTD_ASS8(a, b, c, d, e, f, g, h)       ; i = rhs.i
 #define PTD_ASS10(a, b, c, d, e, f, g, h, i, j    )  PTD_ASS9(a, b, c, d, e, f, g, h, i)    ; j = rhs.j
-#define PTD_ASS11(a, b, c, d, e, f, g, h, i, j, k ) PTD_ASS10(a, b, c, d, e, f, g, h, i, j) ; k = rhs.k
+#define PTD_ASS11(a, b, c, d, e, f, g, h, i, j, k                            ) PTD_ASS10(a, b, c, d, e, f, g, h, i, j)                            ; k = rhs.k
```

Comment:
Having to support so many scalars is making these macros ridiculous.

Reply:
Yeah, this is absurd. Maybe clumping stuff into structs would solve? Or can we create a little helper function which returns a macro with a given number of dims?